Functions/Utilities/New-RsConfigurationSettingObject.ps1

```diff
@@ -4,53 +4,108 @@
 function New-RsConfigurationSettingObject
 {
     <#
-    .SYNOPSIS
-        This script creates a new WMI Object that is connected to Reporting Services WMI Provider.
-
-    .DESCRIPTION
-        This script creates a new WMI Object that is connected to Reporting Services WMI Provider. 
-
-    .PARAMETER SqlServerInstance (optional)
-        Specify the name of the SQL Server Reporting Services Instance.
-    
-    .PARAMETER SqlServerVersion (optional)
-        Specify the version of the SQL Server Reporting Services Instance. 13 for SQL Server 2016, 12 for SQL Server 2014, 11 for SQL Server 2012
-
-    .EXAMPLE 
-        New-RsConfigurationSettingObject 
-        Description
-        -----------
-        This command will retrieve and return WMI Object associated to the default instance (MSSQLSERVER) of SQL Server 2016 Reporting Services.
-
-    .EXAMPLE 
-        New-RsConfigurationSettingObject -SqlServerInstance 'SQL2016' 
-        Description
-        -----------
-        This command will retrieve and return WMI Object associated to the named instance (SQL2016) of SQL Server 2016 Reporting Services.
-
-    .EXAMPLE 
-        New-RsConfigurationSettingObject -SqlServerVersion '11' 
-        Description
-        -----------
-        This command will retrieve and return WMI Object associated to the default instance (MSSQLSERVER) of SQL Server 2012 Reporting Services.
-
-    .EXAMPLE 
-        New-RsConfigurationSettingObject -SqlServerInstance 'SQL2012' -SqlServerVersion '11' 
-        Description
-        -----------
-        This command will retrieve and return WMI Object associated to the named instance (SQL2012) of SQL Server 2012 Reporting Services.
+        .SYNOPSIS
+            This script creates a new WMI Object that is connected to Reporting Services WMI Provider.
+        
+        .DESCRIPTION
+            This script creates a new WMI Object that is connected to Reporting Services WMI Provider.
+        
+        .PARAMETER ReportServerInstance
+            Specify the name of the SQL Server Reporting Services Instance.
+            Use the "Connect-RsReportServer" function to set/update a default value.
+        
+        .PARAMETER ReportServerVersion
+            Specify the version of the SQL Server Reporting Services Instance.
+            Use the "Connect-RsReportServer" function to set/update a default value.
+        
+        .PARAMETER ComputerName
```

Comment:
got a section on that in "about_ReportingServicesTools".
Will push the next commit with that and all the changes required so far either tonight or tomorrow, depending on how fast I can manage it.
Considering switching to Cim-Cmdlets instead of Wmi, but that bears some more testing on. Probably gonna make that my first PR after this bulk is done and out of the way - can do that as a change on a single file after all.